code/src/reducers/todoList.js

```diff
@@ -0,0 +1,34 @@
+import { createSlice } from "@reduxjs/toolkit"
+
+export const todoList = createSlice({
+  name: "todoList",
+  initialState: {
+    items: [],
+    completedItems: []
```

Comment:
you could save some coding by only storing the items in one array and then have a property in each todo to be completed = true or false and filter the items array based on that.